recipes-soc/rdk-gstreamer-utils-platform/rdk-gstreamer-utils-platform.bb

```diff
@@ -0,0 +1,31 @@
+LICENSE = "CLOSED"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
+
+DEPENDS += "${@bb.utils.contains('DISTRO_FEATURES', 'gstreamer1', 'gstreamer1.0  gstreamer1.0-plugins-base', 'gstreamer gst-plugins-base', d)}"
+DEPENDS += "rdk-gstreamer-utils-headers"
+
+SRC_URI = "git://git@github.com/ssadha725_comcast/rdk-gstreamer-utils-raspberrypi;${RDKE_GITHUB_SRC_URI_SUFFIX}"
+SRCREV = "a03aeef1c5978ab3c765307218f1b7b7c25616dd"
+S = "${WORKDIR}/git/"
+
+
+CXXFLAGS += "-I${STAGING_INCDIR}/glib-2.0 -I${STAGING_INCDIR}/gstreamer-1.0 -I${STAGING_DIR_TARGET}/${libdir}/glib-2.0/include/ "
+
+inherit autotools pkgconfig features_check
+
+do_compile () {
+    oe_runmake -C ${S} -f Makefile
+}
+
+do_install() {
+        install -d ${D}/${libdir}
+        install -d ${D}/usr/include
+        install -m 0755 ${S}/librdkgstreamerutilsplatform.so ${D}/${libdir}
+}
+
+
+INSANE_SKIP:${PN} = "dev-so"
+FILES_SOLIBSDEV = ""
+FILES:${PN} += "${libdir}/*.so"
+
+INSANE_SKIP:${PN} += "ldflags textrel"
```

Comment:
Is this needed?

Reply:
**FILES_SOLIBSDEV = ""**
This line ensures that no additional development files are automatically included in the -dev package. By setting this to an empty string, it prevent the build system from automatically including certain files, giving you full control over what gets packaged. This helps avoid conflicts or unexpected inclusions in the -dev package.
FILES:${PN} += "${libdir}/*.so"
This line specifies that all shared library files (*.so) in the ${libdir} directory should be included in the main package (${PN}).
INSANE_SKIP:${PN} += "ldflags textrel"
This line tells the build system to skip certain QA checks for the main package (${PN}).

**By removing these lines would lead to** 
Development files being automatically included in the -dev package, which might not be desired.
Shared library files not being included in any package, causing packaging errors.
QA checks failing due to missing linker flags or text relocations.